bindings/jaxrs/src/main/java/org/commonjava/indy/core/bind/jaxrs/ContentAccessHandler.java

```diff
@@ -172,6 +180,57 @@ public Response doDelete( final String packageType, final String type, final Str
         return doDelete( packageType, type, name, path, eventMetadata, null );
     }
 
+    public Response doDelete( final String packageType, final String type, final String name,
+                              final HttpServletRequest request, EventMetadata eventMetadata )
+    {
+        setContext( PACKAGE_TYPE, packageType );
+
+        if ( !PackageTypes.contains( packageType ) )
+        {
+            ResponseBuilder builder = Response.status( 400 );
+            return builder.build();
+        }
+
+        Set<String> paths;
+        try
+        {
+            paths = mapper.readValue( request.getInputStream(), new TypeReference<Set<String>>() {} );
+        }
+        catch ( IOException e )
+        {
+            logger.error( "Bad request.", e );
+            return responseHelper.formatResponse( e );
+        }
+
+        final StoreType st = StoreType.get( type );
+        StoreKey sk = new StoreKey( packageType, st, name );
+
+        eventMetadata = eventMetadata.set( ContentManager.ENTRY_POINT_STORE, sk );
+        setContext( CONTENT_ENTRY_POINT, sk.toString() );
+
+        Response response;
+        Map<String, String> results = new HashMap<>();
+        for ( final String path : paths )
+        {
```

Comment:
Maybe we need to setContext(PATH, path) here?

Reply:
Sure, that will be helpful.